CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 45.0.0
```

Comment:
Ah, is it more conventional to bump the gem in a separate PR? Will wait to see if other people chime in...